lib/modifiers.js

```diff
@@ -45,7 +45,20 @@ function nestedRestore (arr) {
   const length = arr.length
   for (var i = 0; i < length; i++) {
     const { key, target, value } = arr[i]
-    target[key] = value
+    if (has(target, key)) {
+      target[key] = value
+    }
+    /* istanbul ignore else */
+    if (typeof target === 'object') {
+      const targetKeys = Object.keys(target)
+      for (var j = 0; j < targetKeys.length; j++) {
+        const tKey = targetKeys[j]
+        const subTarget = target[tKey]
```

Comment:
what if subtarget is also an object?

Reply:
Could you pls explain your point here - I dont get it right now
`subTarget` can be an object, and it works imho 

For the example, which is also in the tests (https://github.com/lrecknagel/fast-redact/blob/multi-wildcard-support-finalization/test/index.js#L1228)

```js
const redactOut = fastRedact({
  paths: ['a[*].x.d[*].p.*.i'],
})
const obj = {
  a: [
    { x: { d: [ { i: undefined, j: 'NR' } ] }},
  ]
}
redactOut(obj)
```

we get the following state for subtarget:

![Bildschirmfoto 2022-01-19 um 13 48 31](https://user-images.githubusercontent.com/3778089/150133994-28481115-b7da-459a-a6d7-0f8391f0e0b4.png)